config/routes.rb

```diff
@@ -10,6 +10,8 @@
   # first created -> highest priority.
 
   get "/robots.txt" => RobotsGenerator
+  get "/sitemap.xml.gz" => "sitemap#sitemap", format: :xml
```

Comment:
I'm not sure how search engines work in this case, but I'm thinking do we need to add also `/sitemap.xml` route?